analytics/terraform/spark-k8s-operator/addons.tf

```diff
@@ -62,18 +33,16 @@ module "eks_blueprints_kubernetes_addons" {
   #---------------------------------------
   eks_addons = {
     aws-ebs-csi-driver = {
-      most_recent              = true
       service_account_role_arn = module.ebs_csi_driver_irsa.iam_role_arn
     }
-    coredns = {
-      most_recent = true
-      preserve    = true
-    }
-    kube-proxy = {
+    coredns    = {}
+    kube-proxy = {}
+    vpc-cni = {
```

Comment:
Do we not need the VPC CNI policies to be added to this add-on? Is it a default now?